web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/SelectSecurityGroups.tsx

```diff
@@ -105,11 +113,29 @@ export const SelectSecurityGroups = ({
 
   return (
     <>
+      <Flex alignItems="center" gap={1} mb={2}>
+        <Subtitle3>Select Security Groups</Subtitle3>
+        <ToolTipInfo>
+          <Text>
+            The security groups you pick must meet the following requirements:
+            <ul>
+              <li>
+                all security groups must allow outbound connectivity (eg:
+                0.0.0.0/0) to reach Teleport cluster
+              </li>
+              <li>
+                all security groups must allow inbound connectivity from the ECS
+                task security group or from the ECS task subnet(s) CIDR blocks
+              </li>
+            </ul>
+          </Text>
+        </ToolTipInfo>
+      </Flex>
+
       <P mb={2}>
         Select security groups to assign to the Fargate service that will be
-        running the database access agent. The security groups you pick must
-        allow outbound connectivity to this Teleport cluster. If you don't
-        select any security groups, the default one for the VPC will be used.
+        running the database access agent. If you don't select any security
+        groups, the default one for the VPC will be used.
```

Comment:
Could we make security group selection a requirement instead of optional?

I think it will be better to make the user think about and pick a security group than let them skip an optional step and then likely have to troubleshoot a failing deployment